packages/ui/src/components/Tag/index.tsx

```diff
@@ -106,5 +126,68 @@ export const Tag = ({
       />
     ) : null}
     {isLoading ? <Loader active size={16} /> : null}
-  </StyledContainer>
+  </>
 )
+
+export const Tag = ({
+  children,
+  isLoading,
+  onClose,
+  icon,
+  copiable = false,
+  copyText = 'Copy',
```

Comment:
i don't think we should put default props for translation no ?

Reply:
we should make the props required if copiable is true

Reply:
We already hardcode these in `CopyButton`